examples/Playground.hs

```diff
@@ -49,7 +48,7 @@ main = do
     content <- TIO.readFile "test.toml"
     case parse content of
         Left (ParseException e) -> TIO.putStrLn e
-        Right toml              -> TIO.putStrLn $ prettyToml toml
+        Right tml               -> TIO.putStrLn $ prettyToml tml
```

Comment:
Maybe `toml` is not the best name since if you export `toml` function you can't use `toml` as variable name? Maybe it's better to call it `mkToml` or `declareTolm` or `defineToml` to `constructToml` or `createToml` or something else? What do you think?